### DISCUSSION TITLE
The formatting of the website is completely wrong after pushing

### DISCUSSION THREAD

**Discussion by qyu38** [hexojs/hexo#5413] [Question / Answer]
Hi,
The webpage is looking fine when its hosted through "hexo server"
![image](https://github.com/hexojs/hexo/assets/71168759/ed97a925-0860-4d17-97a8-a6ef1455e02e)
However, after pushing it to the github, the website is looking completely wrong:
![image](https://github.com/hexojs/hexo/assets/71168759/2a50cd9d-ec7b-458a-83c2-b541715a942b)

Did anyone encounter the same issue and could possible advise what could go wrong?

**Comment by stevenjoezhang** [comment]
It's possible that the style file cache hasn't been updated yet. You can wait for a while and then check again.
